Sources/tuist/tuist.docc/Articles/Tuist/Users/installation.md

```diff
@@ -39,7 +39,66 @@ mise use tuist-cloud@latest         # Use the latest tuist-cloud in the current
 mise use -g tuist-cloud@system      # Use the system's tuist-cloud as the global default
 ```
 
-> Tip: We recommend using `mise use` in your Tuist projects to pin the version of Tuist across environments. The command will create a `.tool-versions` file containing the version of Tuist.
+#### Continuous integration
+
+If you are using Tuist in a continuous integration environment, the following sections show how to install Tuist in the most common ones:
+
+##### Xcode Cloud
+
+You'll need a [post-clone](https://developer.apple.com/documentation/xcode/writing-custom-build-scripts#Create-a-custom-build-script) script that installs Mise and Tuist:
+
+```bash
+#!/bin/sh
+curl https://mise.jdx.dev/install.sh | sh
+mise install # Installs the version from .mise.toml
+
+# Runs the version of Tuist indicated in the .mise.toml file
+mise x tuist generate
+```
+
+##### Codemagic
```

Comment:
This is not working for me. The script fails as mise cannot be found.
<img width="733" alt="image" src="https://github.com/tuist/tuist/assets/665916/4a3211b7-6f48-4f08-9a44-1686fbb4bfe9">

Reply:
@jdx does anything come to your mind when you see that permission error? I wonder if it's related to some operation that Mise is trying to do that's not allowed in the session that Xcode creates.

Reply:
unless I'm missing something I don't see a permissions issue. It looks like `~/.local/bin` isn't in PATH.

Reply:
Getting this <img width="718" alt="image" src="https://github.com/tuist/tuist/assets/665916/ce6eb643-0057-49e2-8915-2f9757091a1f"> with 

```sh
# Install Mise + Tuist
curl https://mise.jdx.dev/install.sh | sh

~/.local/bin/mise --version

export PATH="$HOME/.local/bin:$PATH"

mise install # Installs the version from .mise.toml

# Generate the project with tuiss
mise exec tuist -- tuist fetch
mise exec tuist -- tuist generate
```

Reply:
minor thing:

`mise exec tuist -- tuist fetch` you can just do `mise exec -- tuist fetch` or `mise x -- tuist fetch`

Reply:
Would that make a difference regarding the permissions error?

Reply:
no it's just noise

Reply:
Seems to work if I use 

```sh
# Install Mise + Tuist
curl https://mise.jdx.dev/install.sh | sh

~/.local/bin/mise --version

# cd Tuist
~/.local/bin/mise install # Installs the version from .mise.toml

# Generate the project with tuiss
~/.local/bin/mise x -- tuist fetch
~/.local/bin/mise x -- tuist generate
```

Reply:
I think this would as well:

```sh
# Install Mise + Tuist
curl https://mise.jdx.dev/install.sh | MISE_INSTALL_PATH=/usr/local/bin/mise sh

mise --version
mise install # Installs the version from .mise.toml

# Generate the project with tuiss
mise x -- tuist fetch
mise x -- tuist generate
```

And also likely this:

```sh
# Install Mise + Tuist
curl https://mise.jdx.dev/install.sh | MISE_INSTALL_PATH=/usr/local/bin/mise sh
export PATH="$HOME/.local/share/mise/shims:$PATH"

mise --version
mise install # Installs the version from .mise.toml

# Generate the project with tuiss
tuist fetch
tuist generate
```